packages/fiori/src/Bar.js

```diff
@@ -133,6 +133,34 @@ class Bar extends UI5Element {
 			"label": this.design,
 		};
 	}
+
+	resizeHandler(entry) {
+		const bar = entry[0].target;
```

Comment:
When you switch to the ResizeHandler (see comment below), you'll use this.getDomRef() instead of `entry[0].target` here.